FBSimulatorControlTests/Tests/FBSimulatorControlSimulatorLaunchTests.m

```diff
@@ -137,3 +147,49 @@ - (void)testLaunchesMultipleSimulatorsConcurrently
 }
 
 @end
+
+@implementation FBSimulatorControlSimulatorLaunchTests_DefaultSet
+
+- (NSString *)deviceSetPath
+{
+  return nil;
+}
+
+- (void)testLaunchesSingleSimulator
+{
+  [self doTestLaunchesSingleSimulator];
+}
+
+- (void)testLaunchesMultipleSimulatorsConcurrently
+{
+  [self doTestLaunchesMultipleSimulatorsConcurrently];
+}
+
+@end
+
+@implementation FBSimulatorControlSimulatorLaunchTests_CustomSet
+
+- (NSString *)deviceSetPath
+{
+  return [NSTemporaryDirectory() stringByAppendingPathComponent:@"FBSimulatorControlSimulatorLaunchTests_CustomSet"];
+}
+
+- (void)testLaunchesSingleSimulator
+{
+  if (!FBSimulatorControlStaticConfiguration.supportsCustomDeviceSets) {
+    NSLog(@"-[%@ %@] can't run as Custom Device Sets are not supported for this version of Xcode", NSStringFromClass(self.class), NSStringFromSelector(_cmd));
```

Comment:
Good point, I'll add an error condition to `bootSimulator` when a custom device set is attempted to be booted. It should be possible to manipulate a custom device set from Xcode 6, but booting a custom set isn't supported without a shim.